backend/opre_ops/ops_site/portfolios/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from opre_ops.ops_site.cans.models import CommonAccountingNumber
+
+
+class Portfolio(models.Model):
+    """
+    A Portfolio is ___
+    """
+
+    PORTFOLIO_STATUS = [
+        ("In-Process", "In-Process"),
+        ("Not-Started", "Not-Started"),
+        ("Sandbox", "Sandbox"),
+    ]
+
+    name = models.CharField(max_length=120)
+    description = models.TextField(default="", blank=True)
+    status = models.CharField(max_length=30, choices=PORTFOLIO_STATUS)
+    cans = models.ManyToManyField(CommonAccountingNumber)
```

Comment:
I _thought_ I heard that a CAN can only be in one portfolio.  So maybe this should be one-to-many?

Reply:
Good catch, I wasn't sure out of the gate, so just "guessed" for the initial pass.

Reply:
reworked it as a one-to-many